server/Arcadia.Assistant.Calendar.Abstractions/DatesPeriod.cs

```diff
@@ -20,10 +20,10 @@ public sealed class DatesPeriod
 
         public DatesPeriod(DateTime startDate, DateTime endDate, int startWorkingHour = 0, int finishWorkingHour = 8)
         {
-            this.StartDate = startDate;
-            this.EndDate = endDate;
-            this.StartWorkingHour = startWorkingHour;
-            this.FinishWorkingHour = finishWorkingHour;
+            this.StartDate = MinDate(startDate, endDate);
+            this.EndDate = MaxDate(startDate, endDate);
+            this.StartWorkingHour = Math.Min(startWorkingHour, finishWorkingHour);
```

Comment:
lol

Reply:
It is just to be sure that all first/start params are normalized

Reply:
I understand that :)

Reply:
but better to throw an error when created? It's invalid data, after all

Reply:
Somewhere in controller

Reply:
It is checked really, it doesn't allow to create event when Finish is more than Start.

Reply:
what's the purpose of this code then? :) Anyway, better save than sorry?

Reply:
Purpose is to not rely on server handles this case :)